chpl/chpl-auth/src/main/java/gov/healthit/chpl/auth/dto/UserDTO.java

```diff
@@ -111,9 +117,9 @@ public void setTitle(String title) {
         return null;
     }
 
-    public String getName() {
-        return subjectName;
-    }
+//    public String getName() {
+//        return subjectName;
+//    }
 
```

Comment:
No big deal, but do we want to remove this code?

Reply:
I thought about it, but did one of those "find references" things and it showed that some of the jars were looking for it. I have tried testing with it commented out and it seems to be okay, but on the off chance something weird breaks I'd like to have this still here as a reminder. In a week or two, or the next time anyone's in that code, then I think it could be removed.